lib/bpk-component-map/src/BpkMapMarkerStyles.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { StyleSheet } from 'react-native';
+import {
+  colorMonteverde,
+  colorSagano,
+  colorGlencoe,
+  borderRadiusSm,
+  borderRadiusPill,
+  borderSizeSm,
+  colorBlackTint03,
+  colorWhite,
+  spacingBase,
+  spacingMd,
+  spacingLg,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import { shadows } from '../../bpk-styles';
+
+const sharedStyles = {
+  wrapper: {
+    ...shadows.base(),
+    paddingHorizontal: spacingLg / 4, // eslint-disable-line backpack/use-tokens
+    paddingVertical: spacingLg / 4, // eslint-disable-line backpack/use-tokens
```

Comment:
Fun fact, you can trick the linter into allowing this with `spacingLg * 0.25`. I think this code is fine, I just noticed it recently 😛

Reply:
Ah ok, I think we should maybe look at this and how the linter does it for web as I think for this kind of thing on web we allow